mobile/src/bundles/common/components/header/header.tsx

```diff
@@ -0,0 +1,55 @@
+import { type DrawerNavigationProp } from '@react-navigation/drawer';
+import { type ParamListBase } from '@react-navigation/native';
+import React from 'react';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/global-styles';
+
+import { styles } from './styles';
+
+type Header = {
+    navigation: DrawerNavigationProp<ParamListBase>;
+};
+
+const Header = ({ navigation }: Header): JSX.Element => {
+    return (
+        <View
+            style={[
+                styles.header,
+                globalStyles.flexDirectionRow,
+                globalStyles.alignItemsCenter,
+                globalStyles.justifyContentCenter,
+            ]}
+        >
+            <Pressable
+                style={styles.icon}
+                onPress={(): void => {
+                    navigation.openDrawer();
+                }}
```

Comment:
```suggestion
                onPress={navigation.openDrawer}
```

Reply:
Is it ok if I use openDrawe like this, because when I use your suggested variant I can't commit changes (such error: @typescript-eslint/unbound-method)?
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/104503014/8feb95dc-5ab1-4575-9cbf-66f3d6e651e1)

Reply:
Yes, it's ok